doc-www/README.txt

```diff
@@ -0,0 +1,71 @@
+doc-www README.txt
+====================
+
+This doc-www/ directory contains documentation about the `ompi-www` website itself. This stuff
+is for developers who wish to work on maintaining the website, as opposed to people who just
+want to learn about Open MPI.
+
+
+# Hosting Requirements
+
+This section should contain doco describing what sort of web server and libraries etc. you 
+need to run the ompi-www web site.
+
+I'm just guessing here:
+
+* A web server (uh, Apache, I guess?)
+* PHP
+* ...???
```

Comment:
FWIW, when we used to have external sites formally mirror the Open MPI web site, we just told them to have a PHP-enabled web site.  We probably require a "modern" version of PHP these days (i.e., we've almost certainly not taken any effort to restrict our PHP usage to either all-old or all-new PHP stuff -- which is a slightly different way of saying "I don't know what version of PHP is required 😄).

Reply:
Could you check to see which major version of PHP the live server is currently running? I'd like to know if it matches my local setup, and whether I can use modern PHP 7 stuff in our code.

Reply:
```shell
# php --version
PHP 5.4.45 (cli) (built: Apr 17 2017 15:59:08) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.4.0, Copyright (c) 1998-2014 Zend Technologies
    with the ionCube PHP Loader (enabled) + Intrusion Protection from ioncube24.com (unconfigured) v10.2.4, Copyright (c) 2002-2018, by ionCube Ltd.
    with Zend Guard Loader v3.3, Copyright (c) 1998-2013, by Zend Technologies
```

Reply:
Thanks!

PHP 5.4 is [already End of Life](http://php.net/supported-versions.php) and out of current support. Any interest in upgrading to a newer version? 5.6 is in active "security fixes only" support, and 7 is the currently supported major version. I've submitted #208 to fix the PHP 7 incompatibilities.

Reply:
FWIW: I just changed the website to default to php v7.1, which is the highest they support.

Reply:
Excellent.

I hit the site and some PHP 7 warnings are showing up in user-visible pages.

<img width="1097" alt="screen shot 2018-12-15 at 4 04 00 pm" src="https://user-images.githubusercontent.com/2618447/50047447-1c7a3e80-0083-11e9-9a92-a33b7e2a7a5d.png">

Maybe want to prioritize #208?

Reply:
You should probably test it to ensure that is what gets used - the configurator indicates that 7.1 will be used for any file ending in .php, but it doesn't state what happens for non-php files that invoke php functions. The "system default" still shows as 5.4 and cannot be changed.

Reply:
sorry - posts are stepping over each other. I'll let you fix the new warnings 😄

Reply:
#208 committed - holler if you need anything else. Thx for all the effort!

Reply:
Cool. I've updated this PR to reflect that you're using PHP 7 now.

Thanks!